metadata.json

```diff
@@ -69,6 +75,7 @@
     }
   ],
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">=3.2.0 <5.0.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">=3.2.0 <5.0.0"},
+    {"name":"zleslie/pkgng","version_requirement":">=0.2.5 <2.0.0"}
```

Comment:
FWIW, the pkgng provider is set to be included in core. https://github.com/puppetlabs/puppet/pull/3276